doc/glossary/index.rst

```diff
@@ -12,18 +12,18 @@ Glossary
       high-quality images
 
   Cairo
-      The `Cairo graphics <http://cairographics.org>`_ engine
+      The `Cairo graphics <https://cairographics.org>`_ engine
 
 
   dateutil
       The `dateutil <https://dateutil.readthedocs.io>`_ library
       provides extensions to the standard datetime module
 
   EPS
-      Encapsulated Postscript (`EPS <http://en.wikipedia.org/wiki/Encapsulated_PostScript>`_)
+      Encapsulated Postscript (`EPS <https://en.wikipedia.org/w/index.php?title=Encapsulated_PostScript>`_)
```

Comment:
They were throwing a weird warning with linkcheker when I just added https, but they seem to be fine in the actual build like that. I'll change them back and just add https.